home/tests/test_signup.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from django.contrib.auth.models import User
+from home.models import Profile
+
+
+@pytest.mark.django_db
+class TestSignUp:
+
```

Comment:
I think you should also add a test for the form and the template that are displayed.
You can see an example for that test in `test_question_form_and_template_displayed` method in test_insert_question_feature.py file

Reply:
good idea, added.